clang/test/dpct/predefined_macro_replacement.cu

```diff
@@ -173,3 +173,89 @@ int foo2(){
     pl = CUDART_VERSION - major * 1000 - minor * 10;
   }
 }
+
+//CHECK: void foo3() {
+//CHECK-NEXT: #if (SYCL_LANGUAGE_VERSION >= 202001)
+//CHECK-NEXT:   sycl::int2 a1;
+//CHECK-NEXT: #endif
+//CHECK-NEXT: #if (SYCL_LANGUAGE_VERSION >= 202001)
+//CHECK-NEXT:   sycl::int2 a2;
+//CHECK-NEXT: #endif
+//CHECK-NEXT: #define AAA 1
+//CHECK-NEXT: #if (SYCL_LANGUAGE_VERSION >= 202001) && AAA
```

Comment:
will it have better compatibility among different SYCL compiler if using 202000 ?

Reply:
done